manifests/b/BitSum/Coreprio/0.0.6.8/BitSum.Coreprio.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BitSum.Coreprio
+PackageVersion: 0.0.6.8
+PackageLocale: en-US
+Publisher: BitSum

```

```suggestion
Publisher: Bitsum
```
to match Control Panel.